src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Be careful, the bulldozer  started !!!");
+
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Dear employees, you can continue to work, "
+                + "The bulldozer left to refuel");
```

Comment:
Similar to the `doWork` method, the message in `stopWork` should be more informative. Include the type of Machine, which is 'Bulldozer', to make it clear which machine has stopped working. An example message could be: 'Bulldozer has stopped working and is refueling.'